dxr/build.py

```diff
@@ -475,7 +475,7 @@ def index_file(tree, tree_indexers, path, es, index):
             linkses.append(file_to_index.links())
 
             # Per-line stuff:
-            if is_text:
+            if is_text and not islink(path):
```

Comment:
Let's pull the islink() call out of the `for` loop and save some `stat` calls. Also, did you look at the other places in this function where we say `if is_text`? If we don't want to insert LINE needles, regions, and refs for symlinks, we might want to add the islink() condition there, too (and probably factor `islink(…) and is_text` up).